faux-mgs/src/main.rs

```diff
@@ -333,6 +337,9 @@ enum Command {
     ResetComponent {
         #[clap(value_parser = parse_sp_component)]
         component: SpComponent,
+        /// Disables any watchdogs associated with this component reset
```

Comment:
I'm tempted to say this reinforces my concern that the watchdog message should carry a component along with it - this help message seems a little misleading if it only applies to the SP.

I'm also not sure how to word this, exactly; "disable the watchdog" is a thing that internally we _want_ to do after enabling it. Maybe something like "Reset without the automatic rollback safety watchdog"?

Reply:
Rephrased in c5960b41297263f7cff89c378e52b403ca49c8ae

(and yes, there's now a component associated with the reset-with-watchdog message)